hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/service/TestSnapshotDeletingService.java

```diff
@@ -69,7 +69,7 @@ private static Stream<Arguments> testCasesForIgnoreSnapshotGc() {
     return Stream.of(
         Arguments.of(filteredSnapshot, SnapshotInfo.SnapshotStatus.SNAPSHOT_DELETED, true, false),
         Arguments.of(filteredSnapshot, SnapshotInfo.SnapshotStatus.SNAPSHOT_ACTIVE, true, true),
-        Arguments.of(unFilteredSnapshot, SnapshotInfo.SnapshotStatus.SNAPSHOT_DELETED, true, true),
+        Arguments.of(unFilteredSnapshot, SnapshotInfo.SnapshotStatus.SNAPSHOT_DELETED, true, false),
```

Comment:
nit: should we update this test? Earlier `shouldIgnoreSnapshot()` was depended on snapshot's status and SstFilteringServicce's enablement. Now it doesn't depend on `SstFilteringServicce`, we may remove all those cases.

Reply:
we should keep all the tests since we want to ensure this logic doesn't change in future

Reply:
Is keeping negative tests like this a good idea? If we go with this, we may have lots of negative tests rather than actual ones.
I think we should update it but I'll leave it on you.